snap/snapcraft.yaml

```diff
@@ -127,7 +127,7 @@ apps:
       KONG_ADMIN_LISTEN: "0.0.0.0:8001, 0.0.0.0:8444 ssl"
       LC_ALL: C.UTF-8
       LANG: C.UTF-8
-    start-timeout: 15m
+    start-timeout: 7m
```

Comment:
why is this changed here? I don't see anything in the commit message about why this was changed?

Reply:
Nice catch, this should've been cleaned up. I was getting tired of waiting 15 minutes for service failures when debugging my changes. FYI, I have a separate trello card to investigate whether we can finally crank those done to something more reasonable and/or make them configurable.

Reply:
OK, just re-pushed and changed these all back to 15m.